### PR TITLE
Pig Latin Mutation

### PR DESCRIPTION
**Pull request by tralezab**
## About The Pull Request

No pig latin here maintainers, don't ban me

It's a mutation that gives you piglatin!

## Why It's Good For The Game

I had this idea over the freeze as a properly funny way to frick someone's speech up

## Changelog
:cl:
add: Pig Latin mutation
/:cl:

### CODE REVIEW

**Comment by conk444** [comment]
ice-nay, ou-yay ave-hay ood-gay aste-tay. i-way ill-way ove-lay using-way is-thay ew-nay utation-may.

**Comment by tralezab** [comment]
for the untrained:
nice, you have good taste. i will love using this new mutation

File: code/__DEFINES/DNA.dm [hunk 1]
```diff
@@ -57,6 +57,7 @@
 #define UNINTELLIGIBLE /datum/mutation/human/unintelligible
 #define VOID /datum/mutation/human/void
 #define WACKY /datum/mutation/human/wacky
+#define PIGLATIN /datum/mutation/human/piglatin
```
Comment (Mothblocks) [inline code/__DEFINES/DNA.dm:60]: bro my kinda not really abc order!!

**Review by Cyberboss — CHANGES_REQUESTED** [review]
^

**Pull request merged**